apps/webClient/src/views/LetsTalk/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { makeStyles } from "@material-ui/styles";
+import Grid from "@material-ui/core/Grid";
+import Box from "@material-ui/core/Box";
+import FlareIcon from "@material-ui/icons/Flare";
+import Typography from "@material-ui/core/Typography";
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+import Logo from "../../components/Logo/";
+import NavButton from "../../components/NavButton";
+
+interface Props {}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  contrastHeader: {
+    color: "white",
+    textTransform: "uppercase",
+  },
+}));
+
+const LetsTalk: React.FC<Props> = ({}) => {
+  const classes = useStyles();
+
+  return (
+    <Box pt={16} height="100vh">
+      <Grid container justify="center">
+        <Link to="/">
+          <Logo />
+        </Link>
+      </Grid>
+      <Box mt={12}>
+        <Grid container>
+          <Grid
+            item
+            container
+            sm={2}
+            justify="flex-start"
+            alignItems="flex-start"
+          >
+            <NavButton
+              route="/our-work"
+              text="our work"
+              isVertical={true}
+              isLeft={true}
+            />
+          </Grid>
+          <Grid item container sm={8} justify="center">
+            <Grid item>
+              <Typography
+                gutterBottom
+                variant="h5"
+                className={classes.contrastHeader}
+              >
+                let's talk
+              </Typography>
+            </Grid>
+            <Grid item container justify="center">
+              <Box width="50%">
+                <Box>
```

Comment:
I can see some sort of typography function for this text but don't want to over engineer or refactor too much.

Reply:
haha, yeah I have scared you.  I think the contrastHeader and the text probably each deserve their own component.  We can do in this PR or in a follow up.